Adding explanatory text to fit-for-55.md

willemfrishert opened the pull request:
Task of #14: Add English texts to the published infographics

### Preview Build

Current preview build is available at https://en-facts-on-climate--preview-fit-for-55-f27te841.web.app/infographics/fit-for-55

### What does this PR do?

- bulk-translated the Czech text to English using [DeepL](https://www.deepl.com)
- Use [Grammarly](https://www.grammarly.com/) to proofread the texts.
- Note: this is branched off of e04728f281ec716a6a1ff9e96ef74887f2d10167 which introduced the glossary and `co2eq` for #17

### Commented out links:
- Commented out preview-box (using Jekyll comments) as these are (not yet) availtable in English
  - [slug="fit-for-55-opatreni - https://faktaoklimatu.cz/infografiky/fit-for-55-opatreni](https://faktaoklimatu.cz/infografiky/fit-for-55-opatreni)
  - [explainery/emisni-povolenky-ets - https://faktaoklimatu.cz/explainery/emisni-povolenky-ets](https://faktaoklimatu.cz/explainery/emisni-povolenky-ets)

- Commented out link in text (using html comments) as these are (not yet) available in English:
  - [explainery/emisni-povolenky-ets - https://faktaoklimatu.cz/explainery/emisni-povolenky-ets](https://faktaoklimatu.cz/explainery/emisni-povolenky-ets)
  - [explainers/lesnicka-strategie-eu - https://faktaoklimatu.cz/explainery/lesnicka-strategie-eu](https://faktaoklimatu.cz/explainery/lesnicka-strategie-eu)

willemfrishert committed:
Add carbon-neutrality to glossary.yml.

willemfrishert committed:
Translate and proofread the text.

willemfrishert commented on collections/_infographics/policies/fit-for-55.md:
```diff
@@ -1,13 +1,134 @@
+
+#### Transportation
+
+<!--* [Stricter emission regulations for cars and vans](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0556)-->
```
All the eur-lex.europa.eu are easy to update by changing them from https://eur-lex.europa.eu/legal-content/CS to https://eur-lex.europa.eu/legal-content/EN.

However, I tried to use the English URL for this link but 
https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0556 only appears in French.

I searched for this document via https://eur-lex.europa.eu/homepage.html and found the summary in English but it points to the same html page.
On the html page, one is able to navigate to the PDF: https://eur-lex.europa.eu/resource.html?uri=cellar:870b365e-eecc-11eb-a71c-01aa75ed71a1.0001.01/DOC_2&format=PDF

I might need help to locate the English URL for this entry.

tjarnikova replied:
would just using the PDF be sufficient? I think that for whatever reason the HTML just doesn't exist (in any case I tried to find it and could not)

willemfrishert replied:
Yes, I think the PDF should be sufficient but wanted to make sure I did not look into the wrong place for the html version.

tjarnikova commented on collections/_infographics/policies/fit-for-55.md:
```diff
@@ -1,13 +1,134 @@
 ---
 layout:        infographic
-title:         "What is Fit for 55"
+title:         "What is 'Fit for 55'"
```
definitely add a ? "'What is Fit for 55'?" - Czech grammar doesn't require a ? but English does.

willemfrishert replied:
Thanks. Updated.

tjarnikova commented on collections/_infographics/policies/fit-for-55.md:
```diff
@@ -1,13 +1,134 @@
+%}
+{% endcomment %}
+
+['Fit for 55'](https://eur-lex.europa.eu/legal-content/en/TXT/?uri=CELEX%3A52021DC0550), a package of legislative proposals that promotes a **reduction of greenhouse gas emissions by 55% by 2030 compared to 1990**, was presented by the European Commission in July 2021. The package includes proposals amending existing directives or regulations and entirely new legislative proposals, focusing on em ...
```
for clarity, add a "both":
The package includes both proposals amending existing directives or regulations and entirely new legislative proposals, focusing on emissions trading (EU ETS), carbon offsetting at borders, energy and fuels, and land use and forestry.

willemfrishert replied:
Added the word "both" as suggested.

tjarnikova commented on collections/_infographics/policies/fit-for-55.md:
```diff
@@ -1,13 +1,134 @@
+
+['Fit for 55'](https://eur-lex.europa.eu/legal-content/en/TXT/?uri=CELEX%3A52021DC0550), a package of legislative proposals that promotes a **reduction of greenhouse gas emissions by 55% by 2030 compared to 1990**, was presented by the European Commission in July 2021. The package includes proposals amending existing directives or regulations and entirely new legislative proposals, focusing on em ...
+
+The package is currently at the stage of legislative proposals by the European Commission, which are being further discussed in the European Parliament and the Council of the European Union, i.e. with the participation of ministers of the governments of the Member States and national working groups. In this context, the Czech Republic's **presidency of the Council of the EU**,[^presidency-eu], wh ...
```
I suggest we adjust to:

In this context, the Czech Republic's **presidency of the Council of the EU**,[^presidency-eu] in the second half of 2022 is crucial, giving the Czech Republic the opportunity to influence the direction of the negotiations.

willemfrishert replied:
Before I made any correction, I also noticed that the second comma after the reference `[^presidency-eu]` also looked weird.

I updated the text as per your comment.

tjarnikova commented on collections/_infographics/policies/fit-for-55.md:
```diff
@@ -1,13 +1,134 @@
+
+## Context of origin
+
+'Fit for 55' is part of the European Union's (EU) broader strategy to achieve  **<glossary id="carbon-neutrality">Carbon-neutrality</glossary>** by 2050, known as the [**European Green Deal**](https://ec.europa.eu/info/strategy/priorities-2019-2024/european-green-deal).  The European Commission presented the Green Deal in 2019, after which member states agreed on it in 2020. Then, in the summer o ...
```
- in the written text, "carbon neutrality" shouldn't have a hyphen, not sure how that affects the glossary entry. 
- "which the 'Fit for 55' package (loosely translated as "ready for 55%") is designed to ensure." (remove is achieved)

willemfrishert replied:
> - in the written text, "carbon neutrality" shouldn't have a hyphen, not sure how that affects the glossary entry.

I fixed the text here and I also had a look at the glossary entry. It had the same mistake so I fixed it there as well.

> - "which the 'Fit for 55' package (loosely translated as "ready for 55%") is designed to ensure." (remove is achieved)

Corrected this text (i.e., removed 'is achieved')

tjarnikova commented on collections/_infographics/policies/fit-for-55.md:
```diff
@@ -1,13 +1,134 @@
+
+'Fit for 55' is part of the European Union's (EU) broader strategy to achieve  **<glossary id="carbon-neutrality">Carbon-neutrality</glossary>** by 2050, known as the [**European Green Deal**](https://ec.europa.eu/info/strategy/priorities-2019-2024/european-green-deal).  The European Commission presented the Green Deal in 2019, after which member states agreed on it in 2020. Then, in the summer o ...
+
+**To keep global warming below 1.5°C**, relative to pre-industrial levels, **we need to halve greenhouse gas emissions globally** (not just within the EU) by 2030. The European Union (EU-27, excluding the UK) accounted for 7.5% of these emissions in 2020. In absolute terms, EU countries produced 2.6 billion tonnes of CO2 emissions in the same year, ranking third behind China and the US [^emission ...
```
- minor: remove comma after 1.5degC

willemfrishert replied:
Thanks, I removed it.

tjarnikova commented on collections/_infographics/policies/fit-for-55.md:
```diff
@@ -1,13 +1,134 @@
+
+**To keep global warming below 1.5°C**, relative to pre-industrial levels, **we need to halve greenhouse gas emissions globally** (not just within the EU) by 2030. The European Union (EU-27, excluding the UK) accounted for 7.5% of these emissions in 2020. In absolute terms, EU countries produced 2.6 billion tonnes of CO2 emissions in the same year, ranking third behind China and the US [^emission ...
+
+Although the EU and the world are arguably facing the biggest economic and societal transformation in history, according to Eurobarometer data from 2021, the majority of EU citizens (93%) perceive climate change as a serious problem. In the Czech Republic, the figure is over 85%. At the same time, 90% of Europeans and 87% of Czechs agree with the statement, "We should reduce greenhouse gas emissi ...
```
- Change "At the same time," to "In addition,". "at the same time" is typically used to preface a statement of contradiction.

willemfrishert replied:
Thanks! I updated the text.

tjarnikova commented on collections/_infographics/policies/fit-for-55.md:
```diff
@@ -1,13 +1,134 @@
+
+## Principles
+
+The following principles permeate the proposed measures in the 'Fit for 55' package:
```
change "permeate" to "guide" - permeate is a direct translation from CZ that doesn't work contextually.

willemfrishert replied:
Fixed.

tjarnikova commented on collections/_infographics/policies/fit-for-55.md:
```diff
@@ -1,13 +1,134 @@
+* [Carbon offsetting at the border](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0564) (*Carbon Border Adjustment Mechanism*, CBAM)
+* [Revision of the Energy Taxation Directive](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0563) (*Energy Taxation Directive*, ETD)
+
+### Objectives and regulation
```
pluralize to regulations

willemfrishert replied:
Update it it to plural.

tjarnikova commented on collections/_infographics/policies/fit-for-55.md:
```diff
@@ -1,13 +1,134 @@
+
+### Who to contact next
+
+Here are some of the experts who deal with the 'Fit for 55' package:
```
I would write "work with" instead of "deal with" but it is a subjective choice.

willemfrishert replied:
Updated it to "work with" as suggested.

tjarnikova commented on collections/_infographics/policies/fit-for-55.md:
```diff
@@ -1,13 +1,134 @@
+
+[^emission-co2]: Our World in Data: [CO<sub>2</sub> emissions by region](https://ourworldindata.org/co2-emissions#co2-emissions-by-region)
+
+[^eurobarometer]: [Eurobarometer 2021 ](https://climate.ec.europa.eu/system/files/2021-06/cz_climate_2021_en.pdf). The percentage of those who perceive climate change as a serious problem includes respondents who perceive it as a *very* or *somewhat* serious problem. Percentage agreeing with the statement includes respondents who *strongly* or *likely* agree with the statement.
```
Add "The" in front of the second percentage. "The percentage agreeing with"...

willemfrishert replied:
Thanks. Updated the article here.

tjarnikova reviewed (changes requested):
Small edits for legibility, none major.

willemfrishert committed:
Update title.

willemfrishert committed:
Correct text in first paragraph.

willemfrishert committed:
Correct text in second paragraph.

willemfrishert committed:
Correct 'Carbon neutrality'.

Remove hyphen.

willemfrishert committed:
Update paragraphs concerning 'Context of origin'.

willemfrishert committed:
Correct text in 'Principles' section.

willemfrishert committed:
Update section to 'Objectives and regulations'.

willemfrishert committed:
Correct resource reference link.

Updating the link to the PDF in English as the original link for English (https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0556) resolves to the text in article French.

willemfrishert committed:
Update text in 'Resources and notes'.
